cpp/include/raft/random/rng.cuh

```diff
@@ -170,7 +199,70 @@ void normalInt(const raft::handle_t& handle,
  *
  * Each row in this table conforms to a normally distributed n-dim vector
  * whose mean is the input vector and standard deviation is the corresponding
- * vector or scalar. Correlations among the dimensions itself is assumed to
+ * vector or scalar. Correlations among the dimensions itself are assumed to
+ * be absent.
+ *
+ * @tparam OutputValueType data type of output random number
+ * @tparam IndexType data type used to represent length of the arrays
+ *
+ * @param[in] handle raft handle for resource management
+ * @param[in] rng_state random number generator state
+ * @param[in] mu_vec mean vector (of length `out.extent(1)`)
+ * @param[in] sigma_vec Either the standard-deviation vector
+ *            (of length `out.extent(1)`) of each component,
+ *            or a scalar standard deviation for all components.
+ * @param[out] ptr the output table
```

Comment:
I think this is intended to be `@param[out] out` now, right?

Reply:
@cjnolet Oh oof, that's right!  I need to get ready for a meeting, but I can fix this either here or in PR #897 -- please let me know which you prefer -- thanks!

Reply:
No problem! I went ahead and just made the change since it's so tiny. Both PRs look great so since the changes are already in #897, I think it's fine to just merge that one over.